cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_ionic_client_app/templates/src/app/services/shared/business-operator.service.ts.ftl

```diff
@@ -38,6 +38,6 @@ export class BusinessOperatorService {
    * @returns The url to the ${variables.etoName?lower_case} management service.
    */
   ${variables.etoName?lower_case}Service() {
-    return this.restPath + '${variables.component?uncap_first}/v1/${variables.etoName?lower_case}/';
```

Comment:
why has this being changed? this seems to me an invalid change as it fixes the service URL to some demo one.

Reply:
Yes, it is by mistake, corrected it .